04-concept-api/references/attribute_type.yml

```
@@ -70,7 +70,7 @@ methods:
   - method:
     common: &method-getInstances
       title: Retrieve all instances of this AttributeType
-      description: Retrieves all Attributes that are instances of this Type.
+      description: Retrieves all direct and indirect Attributes that are instances of this Type.
```

This change also applies in 7 other places in this PR
```suggestion
      description: Retrieves all Attributes that are direct and indirect instances of this Type.
```

---

all done!